canonical-web-and-design/maas-ui
# Migrate to TypeScript.

## Done
* Add TypeScript compiler, targeting ES6.
* Add eslint overrides to allow for separate eslint configuration for js and ts files (this requires the explicit addition of libraries provided by CRA)
* Convert a simple utility function to TypeScript as a test (capitaliseFirst).

## QA
* Ensure app runs, and builds as expected.

This might be a good time to ensure your IDE works well with TypeScript and they you see inline errors, have intellisense etc.
